.github/workflows/ci.yml

```diff
@@ -30,18 +30,9 @@ jobs:
       - name: Clone This Repo
         uses: actions/checkout@v3
       - name: Link Verification
-        uses: FreeRTOS/CI-CD-Github-Actions/link-verifier@v2
-
-  url-check:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
+        uses: FreeRTOS/CI-CD-Github-Actions/link-verifier@main
```

Comment:
This was incorrectly pointed to the v2 branch. Pointing back to mainline as part of this PR.